snapshot/snapshot.go

```diff
@@ -254,27 +254,17 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 		return nil, err
 	}
 
-	logCtx.Infof("Preparing. Key=%s, Parent=%s, Labels %v", key, parent, base.Labels)
+	logCtx.Infof("Preparing snapshot with labels %v", base.Labels)
 	if target, ok := base.Labels[label.TargetSnapshotRef]; ok {
-		// check if image layer is nydus layer
+		// check if image layer is nydus data layer
 		if o.fs.Support(ctx, base.Labels) {
-			logCtx.Infof("nydus data/meta layer, skip download and unpack %s", key)
-			_, isBootstrapLayer := base.Labels[label.NydusMetaLayer]
-			if isBootstrapLayer {
-				err = o.fs.PrepareMetaLayer(ctx, s, base.Labels)
```

Comment:
Some confusion, why not prepare the meta layer?

Reply:
The bootstrap file is handled by containerd.
And the `o.fs.Support()` checks `labels[label.NydusDataLayer]`, so it filters out bootstrap files too.

Reply:
Ok, that make sense.